qa/rpc-tests/listsinceblock.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+# Copyright (c) 2014-2017 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import *
```

Comment:
Nit: It is sufficient to only import what you need. (E.g. `assert_equal`)

Also, `+x` is missing on the permissions of this file.